results/models.py

```diff
@@ -11,5 +11,13 @@ class QuestionResult(models.Model):
 
 class FinishedAssignment(models.Model):
     assignment = models.ForeignKey(assignments.models.Assignment, related_name='assignment')
-    finished = models.DateTimeField(null=True)
-    answers = models.ManyToManyField(assignments.models.Answer)
\ No newline at end of file
+    answer = models.ManyToManyField(assignments.models.Answer, related_name='answersToAssignment')
+    passed = models.BooleanField(null=False, default=False)
+    score = models.PositiveIntegerField(null=False, default=0)
```

Comment:
`passed` kan regnes ut ved å sjekke `self.score >= self.assignment.passingGrade`.
`score` kan regnes ut ved å summere `answer.isCorrect` for alle answers som er i mange-til-mange relasjonen.